package.json

```diff
@@ -33,6 +33,7 @@
 		"test:unit": "DOTENV_CONFIG_PATH=.env.test jest -c jest.config.unit.ts"
 	},
 	"dependencies": {
+		"@figma/rest-api-spec": "^0.33.0",
```

Comment:
I replaced some of our custom types with types from our [official rest api spec](https://github.com/figma/rest-api-spec) where possible